newIDE/app/src/InstancesEditor/FullSizeInstancesEditorWithScrollbars.js

```diff
@@ -62,7 +62,11 @@ export default class FullSizeInstancesEditorWithScrollbars extends Component<
   componentDidMount() {
     if (this._editor) {
       this._handleViewPositionChange(this._editor.getViewPosition());
-      this._editor.scrollTo(this.props.options.xValue, this.props.options.yValue);
+      // $FlowFixMe
```

Comment:
Couldn't find a way to fix this, not sure if this is correct

Reply:
Flow is surely erroring because `this._handleViewPositionChange` could maybe have been changing `this._editor` to null (yes, unlikely.. but could happen).
The solution is to use a variable for using this._editor, guaranteeing it won't be nulled:

```
const editor = this._editor;
if (editor) {
  this._handleViewPositionChange(...);
  editor.scrollTo(...)
}
```